perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh

```diff
@@ -0,0 +1,15 @@
+#i!/bin/bash
+set -e
+sudo apt-get update
+echo Installing fio
+sudo apt-get install fio -y
+echo Installing gcsfuse
+curl -L -O https://github.com/GoogleCloudPlatform/gcsfuse/releases/download/v0.41.1/gcsfuse_0.41.1_amd64.deb
```

Comment:
please make the GCSFuse version a named string and use it everywhere. Will make updating the version easier

GCSFUSE_VERSION=v0.41.1

download/$GCSFUSE_VERSION/..

Reply:
done